dump/mapping/bufr_sfcsno.py

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env python3
+import sys
+import os
+import argparse
+import time
+import numpy as np
+import bufr
+from pyioda.ioda.Engines.Bufr import Encoder as iodaEncoder
+from bufr.encoders.netcdf import Encoder as netcdfEncoder
+from wxflow import Logger
+
+# Initialize Logger
+# Get log level from the environment variable, default to 'INFO it not set
+log_level = os.getenv('LOG_LEVEL', 'INFO')
+logger = Logger('BUFR2IODA_sfcsno.py', level=log_level, colored_log=False)
+
+
+def logging(comm, level, message):
```

Comment:
We might want to make a logging module that we can import, so that we do not need to redefine this function in every **py** mapping file. 

We don't have to do this now.

Reply:
I will do this in my next PR.